tester/orb.yaml

```diff
@@ -225,7 +225,7 @@ jobs:
             LD_LIBRARY_PATH: /usr/local/lib
       - run:
           working_directory: <<parameters.cwd>>
-          command: docker-compose <<parameters.compose_args>> logs
+          command: docker-compose <<parameters.compose_args>> logs -t | sort -u -k 3
```

Comment:
The `-u` flag confuses me a bit -- where is the duplication coming from? I imagine if a container logged the same thing twice, we may want to know.

Reply:
good point